src/axom/sidre/core/Group.hpp

```diff
@@ -987,6 +1039,16 @@ class Group
    */
   const MapCollection<Group>* getNamedGroups() const;
 
+  /*!
+   * \brief Method to (recursively) accumulate information about data in
+   *        a Group subtree.
+   *
+   * \sa getDataInfo
```

Comment:
This should have doxygen descriptions of the parameters. Though looking at this file not many other functions do this...

Reply:
Added.